compiler/passes/errorHandling.cpp

```diff
@@ -537,7 +537,7 @@ static void printReason(BaseAST* node, implicitThrowsReasons_t* reasons)
       if (calledFn->throwsError()) {
         if (reasons->count(calledFn)) {
           BaseAST* reason = (*reasons)[calledFn];
-          USR_PRINT(reason, " is reason function throws");
+          USR_PRINT(reason, "is reason function throws");
```

Comment:
I might extend this somewhat so that the message reads: "note: this line is the reason the function throws" or something along those lines.  It was a bit too curt for my taste.

Reply:
how about `reason function throws here` ? Or `called throwing function here` ?

Reply:
Of those two options, I would prefer `called throwing function here`.  My problem with the other one and the old version is that the message seems a little passive and vague.  `called throwing function here` seems clearer to me

Reply:
I'm currently proceeding with `call to throwing function here`. thanks!